src/librustc_codegen_llvm/llvm_util.rs

```diff
@@ -184,6 +184,8 @@ const POWERPC_WHITELIST: &[(&str, Option<Symbol>)] = &[
     ("power8-vector", Some(sym::powerpc_target_feature)),
     ("power9-vector", Some(sym::powerpc_target_feature)),
     ("vsx", Some(sym::powerpc_target_feature)),
+    ("secure-plt", Some(sym::powerpc_target_feature)),
+    ("msync", Some(sym::powerpc_target_feature)),
```

Comment:
FWIW these features are typically just those mentioned in `#[cfg(target_feature = "...")]` in code, so if you don't need to mention it in code (and just need to pass it on the CLI) then this change may not be necessary

Reply:
I will revert these changes, I thought they are necessary.

Reply:
The changes have been reverted:
https://github.com/BaoshanPang/rust/commit/62c0fbd4e947240dd4b4de0fb3ba63255e4260c3